reference-implementation/lib/ReadableByteStreamController-impl.js

```diff
@@ -38,6 +39,13 @@ exports.implementation = class ReadableByteStreamControllerImpl {
       throw new TypeError(`The stream (in ${state} state) is not in the readable state and cannot be closed`);
     }
 
+    if (this._pendingPullIntos.length > 0) {
```

Comment:
Why locate these checks here, instead of e.g. in ReadableByteStreamControllerClose step 4?

Reply:
Good point, I'll move things around a bit. 👍

Currently, this also prevents valid usages like:
```javascript
const { done, value } = await transferAndFillViewSomehow(c.byobRequest.view);
if (done) {
  c.close(); // would throw when implemented as currently written
  c.byobRequest.respondWithNewView(value);
}
```
so the check *definitely* needs to move. 🙂

Reply:
Done. I removed this check, and added tests to ensure we can call `close()` and afterwards respond with a *transferred* view (see web-platform-tests/wpt@9b7f4358098f0aea35eb8de8333ec5b6b6f46446).